pydatastructs/graphs/algorithms.py

```diff
@@ -182,41 +186,45 @@ def _generate_layer(**kwargs):
         status = status and operation(curr_node, "", *op_args, **op_kwargs)
     return status
 
-def _breadth_first_search_parallel_adjacency_list(
-    graph, source_node, num_threads, operation, *args, **kwargs):
+
+def _breadth_first_search_parallel_adjacency_list(graph, source_node, num_threads, operation, *args, **kwargs):
     visited, layers = {}, {}
     layers[0] = set()
     layers[0].add(source_node)
     visited[source_node] = True
     layer = 0
     while len(layers[layer]) != 0:
-        layers[layer+1] = set()
+        layers[layer + 1] = set()
         with ThreadPoolExecutor(max_workers=num_threads) as Executor:
             for node in layers[layer]:
                 status = Executor.submit(
-                         _generate_layer, args=
-                         (graph, node, layers[layer+1], visited,
-                          operation, *args), kwargs=kwargs).result()
+                    _generate_layer, args=(graph, node, layers[layer + 1], visited, operation, *args),
+                    kwargs=kwargs).result()
         layer += 1
         if not status:
             return None
 
+
 _breadth_first_search_parallel_adjacency_matrix = _breadth_first_search_parallel_adjacency_list
 
+
 def _generate_mst_object(graph):
     mst = Graph(*[getattr(graph, str(v)) for v in graph.vertices])
     return mst
 
+
 def _sort_edges(graph, num_threads=None):
     edges = list(graph.edge_weights.items())
     if num_threads is None:
         sort_key = lambda item: item[1].value
         return sorted(edges, key=sort_key)
 
+    # noinspection PyTypeChecker
```

Comment:
Is this one also intentional?

Reply:
no, it's just for ignoring error but now that has been resolved so no need of it removed that was made by flake8 cli when it was resolving